### ISSUE TITLE
[MU4 Issue] Outlining text

### ISSUE THREAD

**Issue by Grim-O** [musescore/MuseScore#14340]
Outlining any kind of text to one side actually outlines it to the other side.
e.g.: outlining dynamics to the right moves the dynamic marking to the left.
![Screenshot (8)](https://user-images.githubusercontent.com/115939617/200126040-5a343977-a1ee-4d18-a1f8-2f664efa7c84.png)

**Comment by DaddyLudwig** [comment]
I think you mean align, not outline.

**Comment by Grim-O** [comment]
yes, is that not synonymous?

**Comment by DaddyLudwig** [comment]
No, outline means like you're writing the structure of something, align is what you're talking about with lining up the text.

**Comment by MarcSabatella** [comment]
And as far as I can tell, this is behaving correctly.  Right-align means, the right edge of the text is aligned with the note it is attached to.  That does indeed mean moving it to the left, not sure what you're expecting?

**Comment by Grim-O** [comment]
perhaps it's logical to you because you're used to it but in any other text engine aligning to the right means the text aligns with the right edge. now it's just very confusing that a symbol which looks like the text will move to the right actually moves the text to the left.

**Comment by MarcSabatella** [comment]
The alignment is for the *text*, not the *note*.  The *right edge of the text" is what is aligned.  Indeed, it could be useful someday to also add a control over which edge of the note the text aligns to, but that would be a totally separate thing.  This control is to control which edge of the text aligns.  And it's consistent with how other programs do this.  Could be a language issue confusing things, I guess.

**Comment by cbjeukendrup** [comment]
@Grim-O With single line text, it may be a little bit confusing, but as you can see with multi-line text, it behaves exactly as in other apps:

https://user-images.githubusercontent.com/48658420/200131598-4a262556-0b3e-4dc4-a578-321f4c48aaa1.mov

The only difference is that MuseScore (of course) aligns text relative to a _point_ instead of relative to a _box_, because in MuseScore's case there is no natural choice for a box.

However, if you really want full control, you would need two settings: one to control the alignment of the whole text block relative to the anchor point, and one to control the alignment of the multiple lines relative to each other.

**Comment by Grim-O** [comment]
ok I understand this but then why does the order of the symbols imply a box?

**Comment by Tantacrul** [comment]
Simple answer is to just show a box outline when text is selected. That would clear up the behaviour.

Then, if we just stop the text moving around ridiculously and keep the alignment only to its own bounding box, it will be FAR easier to use and predict. At present, the behaviour is just nonsense.

**Comment by rpatters1** [comment]
Does this issue apply to text in frames as well? I can't see any way to have, for example, footer text that is left or right aligned but center justified. (For reference, this is easily accomplished in other programs.) Before writing it up, I want to be check if this issue covers it.
